platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -248,12 +274,14 @@ private MigrationReport doImport(
       SecurityLogger.audit("Warnings importing configuration settings from file {}", exportFile);
       // don't leave the zip file there if the import succeeded
       deleteQuietly(exportFile.toFile());
+      deleteQuietly(keyFile.toFile());
       report.record(new MigrationWarning(Messages.IMPORT_SUCCESS_WITH_WARNINGS, exportFile));
       report.record(new MigrationWarning(Messages.RESTART_SYSTEM_WHEN_WARNINGS));
     } else {
       SecurityLogger.audit("Exported configuration settings from file {}", exportFile);
       // don't leave the zip file there if the import succeeded
       deleteQuietly(exportFile.toFile());
+      deleteQuietly(keyFile.toFile());
```

Comment:
❓ How about the checksum file?

Reply:
it gets removed now on both failure and success